canonical-websites/www.ubuntu.com
# Add CSS switch to allow development on Vanilla v1

## Done

Added mechanism to switch a template to use Vanilla Brochure Theme. This involves quite a bit of duplication of files and folders for now but we will clean this all up at the end.

## QA

- Pull code
- Run `./run start --watch`
- Go to [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about) and confirm it looks like live
- Open in your editor: `templates/about/index.html`
- Read comment in line 1
- Change include name as instructed
- Refresh [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about)
- That page should now pull v1 CSS
- Check no other pages affected
